dstep/translator/Context.d

```diff
@@ -33,8 +34,12 @@ class Context
     private TypedefIndex typedefIndex_ = null;
     private Translator translator_ = null;
     private Output globalScope_ = null;
+    private Cursor[string] typeNames_;
 
-    public this(TranslationUnit translUnit, Options options, Translator translator)
+    public MacroDefinition[string] macroDefinitions;
+    string macroLinkage = "extern (D)";
```

Comment:
The whole file is wrapped to `extern (C):`, so I thought it would be wise to override it for the translations of macros to not cause a clash with other symbols (e.g. from static library, mangled in C-style).

> Why isn't the space included from the beginning?

I thought about a CLI switch to control that behavior and I do not want the user to be obliged to remember about formatting issues.